cadence/contracts/TransactionGenerationUtils.cdc

```diff
@@ -198,6 +235,11 @@ pub contract TransactionGenerationUtils {
         return StringUtils.join(res, "\n")
     }
 
+    pub fun getAddressFromType(_ type: Type): String {
+        let typeStr = Type<@FlowUtilityToken.Vault>().identifier
```

Comment:
I think this is supposed to use the `type` rather than hardcoding flowutilitytoken.vault ?